entity-framework/core/providers/sqlite/built-in-functions.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Built-in Functions - SQLite Database Provider - EF Core
+description: Built-in Functions in the SQLite EF Core database provider
+author: bricelam
+ms.date: 10/05/2020
+uid: core/providers/sqlite/built-in-functions
+---
+# Built-in Functions in the SQLite EF Core Provider
+
+This table shows which .NET members are translated into which SQL functions when using the SQLite provider.
+
+.NET                                      | SQL                                                                      | Added in
+----------------------------------------- | ------------------------------------------------------------------------ | --------
+bytes.Contains(value)                     | instr(@bytes, char(@value)) > 0                                          | EF Core 5.0
+bytes.Length                              | length(@bytes)                                                           | EF Core 5.0
+DateTime.Now                              | datetime('now', 'localtime')
+DateTime.Today                            | datetime('now', 'localtime', 'start of day')
+DateTime.UtcNow                           | datetime('now')
+dateTime.AddDays(value)                   | datetime(@dateTime, @value \|\| ' days')
```

Comment:
```suggestion
DateTime.AddDays(value)                   | datetime(@dateTime, @value \|\| ' days')
```

Reply:
I used lower-case (and the suffix *value*) to represent instance members. Syntax highlighting would also help reinforce that, but I’d have to add it manually since these can’t be code blocks.